tsconfig.json

```diff
@@ -1,37 +1,24 @@
 {
 	"compilerOptions": {
-		"allowSyntheticDefaultImports": true,
-		"alwaysStrict": true,
-		"declaration": true,
-		"emitDecoratorMetadata": true,
-		"experimentalDecorators": true,
+		"outDir": "dist",
+		"rootDir": "src",
+		"module": "esnext",
+		"lib": ["esnext",],
+		"target": "es6",
+		"moduleResolution": "node",
+		"esModuleInterop": true,
 		"strict": true,
+		"noImplicitReturns": true,
+        "noImplicitThis": true,
+        "noUnusedLocals": true,
+		"noFallthroughCasesInSwitch": true,
+		"noUnusedParameters": true,
+		"forceConsistentCasingInFileNames": true,
 		"incremental": true,
-		"lib": [
-			"esnext",
-			"esnext.array",
-			"esnext.asynciterable",
-			"esnext.intl",
-			"esnext.symbol"
-		],
-		"module": "commonjs",
-		"moduleResolution": "node",
-		"noUnusedLocals": true,
-		"outDir": "./dist",
-		"pretty": true,
 		"removeComments": true,
-		"resolveJsonModule": true,
+		"declaration": true,
 		"sourceMap": true,
-		"strictFunctionTypes": true,
-		"strictBindCallApply": true,
-		"target": "esnext",
-		"types": ["node"]
 	},
-	"exclude": [
```

Comment:
why was this removed

Reply:
Exclusions are only relevant when you want to exclude something that was previously included. i.e. if you have the "tests" folder under "src" or keep some legacy code in there. 

TL;DR: Unless you need to filter what is included, exclusions aren't necessary.